operations/reservations.md

```diff
@@ -289,222 +290,289 @@ Returns all reservations specified by any identifier, customer or other filter.
+| `Notes` | array of [Service order note](serviceordernotes.md#service-order-note) | optional | Notes of the reservations. |
+| `QrCodeData` | array of [Reservation QR code data](reservations.md#reservation-qr-code-data) | optional | QR code data of the reservations. |
+| `Companies` | array of [Company](companies.md#company) | optional | Companies related to the reservations. |
+| `ResourceAccessTokens` | array of [Resource access token](resourceaccesstokens.md#resource-access-token) | optional | Resource access tokens for the reservations. |
+| `Cursor` | string | optional | Unique identifier of the last and hence oldest reservation returned. This can be used in [Limitation](../guidelines/pagination.md#limitation) in a subsequent request to fetch the next batch of older reservations. |
 
 #### Reservation (ver 2017-04-12)
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Id` | string | required | Unique identifier of the reservation. |
-| `ServiceId` | string | required | Unique identifier of the [Service](services.md#service) that is reserved. |
-| `GroupId` | string | required | Unique identifier of the [Reservation group](#reservation-group). |
+| `ServiceId` | string | required | Unique identifier of the Service that is reserved. |
+| `GroupId` | string | required | Unique identifier of the Reservation group. |
 | `Number` | string | required | Confirmation number of the reservation in Mews. |
-| `ChannelNumber` | string | optional | Number of the reservation within the Channel \(i.e. OTA, GDS, CRS, etc\) in case the reservation group originates there \(e.g. Booking.com confirmation number\). |
+| `ChannelNumber` | string | optional | Number of the reservation within the Channel (i.e. OTA, GDS, CRS, etc) in case the reservation group originates there (e.g. Booking.com confirmation number). |
 | `ChannelManagerNumber` | string | optional | Unique number of the reservation within the reservation group. |
 | `ChannelManagerGroupNumber` | string | optional | Number of the reservation group within a Channel manager that transferred the reservation from Channel to Mews. |
-| `ChannelManager` | string | optional | Name of the Channel manager \(e.g. AvailPro, SiteMinder, TravelClick, etc\). |
-| `State` | string [Reservation state](#reservation-state) | required | State of the reservation. |
-| `Origin` | string [Reservation origin](#reservation-origin) | required | Origin of the reservation. |
-| `OriginDetail`| string | optional | Details about the [reservation origin](#reservation-origin). |
-| `Purpose` | string [Reservation purpose](#reservation-purpose) | optional | Purpose of the reservation. |
+| `ChannelManager` | string | optional | Name of the Channel manager (e.g. AvailPro, SiteMinder, TravelClick, etc). |
+| `State` | string | required | State of the reservation. |
+| `Origin` | string | required | Origin of the reservation. |
```

Comment:
These should be enums.

Reply:
Actually this one is sort of weird. Looking at the serializer, it seems to concatenate reservation origin with commander origin, so you could get, for example, `CommanderInPerson`…